reference/7.0/Microsoft.PowerShell.Core/About/about_Calculated_Properties.md

```diff
@@ -0,0 +1,506 @@
+---
+Locale: en-US
+ms.date: 08/07/2020
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_calculated_properties?view=powershell-7&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_Calculated_Properties
+---
+# About Calculated Properties
+
+## Short Description
+
+Calculated properties allow you to dynamically add properties and formatting to
+objects in the pipeline.
+
+## Long Description
+
+PowerShell provides several cmdlets that operate on properties of objects.
```

Comment:
This reads kind of strangely as written. 
- Two consecutive sentences that start the exact same way:
  - "PowerShell provides...PowerShell Provides..."
- The first sentence as written, seems like subject is about cmdlets that operate on properties and not necessarily speaking specifically about Calculated Properties.This entire section could probably be written something like this:

PowerShell provides several cmdlets that can operate on properties of objects using a method to create Calculated Properties. Calculated Properties are used to dynamically create new properties that are added to the output object or provide custom formatting for an existing property.

I removed the sentence, "Calculated Properties are dynamically created at run time" because as written, it doesn't really add any useful information and is more confusing than anything else. If It would remove critical information to take it out, it should be clarified and simplified to express that information. Or, be followed by a clarifghying sentence that adds more info?

Reply:
I reworked that paragraph